core/src/main/kotlin/io/github/chrislo27/rhre3/editor/Editor.kt

```diff
@@ -913,6 +913,71 @@ class Editor(val main: RHRE3Application, stageCamera: OrthographicCamera, attach
                             // Export screen
                             main.screen = ExportRemixScreen(main)
                         }
+                    } else if (Gdx.input.isKeyJustPressed(Input.Keys.F)) {
+                        // Select following
+                        val selectionMinX: Float = this.selection.minBy { it.bounds.x }?.bounds?.x ?: remix.playbackStart
+                        val newSelection = remix.entities.toList().filter { it.bounds.x >= selectionMinX }
+                        remix.mutate(EntitySelectionAction(this, this.selection, newSelection))
+                    } else if (Gdx.input.isKeyJustPressed(Input.Keys.R)) {
+                        // Select preceding
+                        val selectionMaxX: Float = this.selection.maxBy { it.bounds.maxX }?.bounds?.maxX ?: remix.playbackStart
+                        val newSelection = remix.entities.toList().filter { it.bounds.maxX <= selectionMaxX }
+                        remix.mutate(EntitySelectionAction(this, this.selection, newSelection))
+                    } else if (Gdx.input.isKeyJustPressed(Input.Keys.T)) {
+                        // Select between
+                        val selectionMinX: Float = this.selection.minBy { it.bounds.x }?.bounds?.x ?: 0f
+                        val selectionMaxX: Float = this.selection.maxBy { it.bounds.maxX }?.bounds?.maxX ?: 0f
+                        val newSelection = remix.entities.toList().filter { it.bounds.x >= selectionMinX && it.bounds.maxX <= selectionMaxX }
+                        remix.mutate(EntitySelectionAction(this, this.selection, newSelection))
+                    } else if (Gdx.input.isKeyJustPressed(Input.Keys.C)) {
+                        // Copy
+                        Gdx.app.clipboard.contents = PatternStoreScreen.entitiesToJson(remix, selection.toList())
+                    } else if (Gdx.input.isKeyJustPressed(Input.Keys.X)) {
+                        // Cut
+                        Gdx.app.clipboard.contents = PatternStoreScreen.entitiesToJson(remix, selection.toList())
+                        
+                        remix.entities.removeAll(this.selection)
+                        remix.addActionWithoutMutating(ActionGroup(listOf(
+                                EntityRemoveAction(this, this.selection,
+                                                   this.selection.map { Rectangle(it.bounds) }),
+                                EntitySelectionAction(this, this.selection.toList(), listOf())
+                                                                         )))
+                        this.selection = listOf()
+
+                        updateMessageLabel()
+                    } else if (Gdx.input.isKeyJustPressed(Input.Keys.V)) {
```

Comment:
Pasting without explicitly dragging is something I have avoided. Note that so far, copying is holding ALT and clicking and dragging on a selection. I will need to review and consider this further.